.circleci/config.yml

```diff
@@ -49,5 +49,6 @@ workflows:
             - cimg/ruby:3.1
             - cimg/ruby:3.2
             - circleci/jruby:9.1
```

Comment:
nit: would be handy to leave a small comment on why only 9.1 is kept as is.

Reply:
Good point! Done.